cli/src/main/java/bio/terra/tanagra/cli/utils/Context.java

```diff
@@ -9,12 +9,10 @@ public final class Context {
   private static final String LOG_FILENAME = "tanagra.log";
   public static final String IS_TEST = "IS_TEST";
   private static final String IS_TEST_TRUE_VALUE = "true";
-
```

Comment:
Please try to keep these types of sweeping changes to unrelated files out of PRs with actual changes. It makes them harder to review and can be painful in the event of rollbacks or cherry picks.

Reply:
noted